validator/src/main/resources/validations/java/ec2/asg/log-validation.yml

```diff
@@ -22,4 +22,7 @@
   httpMethod: "get"
   callingType: "http"
   expectedLogStructureTemplate: "JAVA_EC2_ASG_CLIENT_CALL_LOG"
-
+-
+  validationType: "cw-log"
+  callingType: "none"
+  expectedLogStructureTemplate: "JAVA_RUNTIME_METRIC_LOG"
```

Comment:
We need to add these new templates to the PR

Reply:
Oh they were already added here: https://github.com/aws-observability/aws-application-signals-test-framework/pull/282/files